### PR TITLE
Add e2e tests for AWS

### PR DESCRIPTION
**Pull request by katexochen**

### CODE REVIEW

File: .github/actions/constellation_create/az-logs.sh [hunk 1]
```diff
@@ -16,6 +16,6 @@ for scaleset in $scalesetslist; do
         bloburi=$(az rest --method post --url https://management.azure.com/subscriptions/${subscription}/resourceGroups/${1}/providers/Microsoft.Compute/virtualMachineScaleSets/${scaleset}/virtualmachines/$instanceid/retrieveBootDiagnosticsData?api-version=2022-03-01 | jq '.serialConsoleLogBlobUri' -r)
         sleep 4
         curl -sL -o "./${scaleset}-${instanceid}.log" $bloburi
-        echo $(realpath "./${scaleset}-${instanceid}.log")
+        realpath "./${scaleset}-${instanceid}.log"
```
[the remark this answers is not in the document]
Comment (datosh) [reply .github/actions/constellation_create/az-logs.sh:19]: No sure if this goes to far, but [`${{ runner.debug }}`](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context) exposes whether this build was re-run using the GitHub action debug option. Then we could reduce on debug output on normal runs.

**Review by datosh — APPROVED** [review]
LGTM, only two nits.

File: .github/actions/constellation_create/az-logs.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (katexochen) [reply .github/actions/constellation_create/az-logs.sh:19]: This is done in #514 .

> Then we could reduce on debug output on normal runs.

If the output is to much on the first look, we can just define a output group that is closed by default. Not sure this is needed in the above case.

File: .github/actions/constellation_create/action.yml [hunk 2]
```diff
@@ -79,6 +86,14 @@ runs:
             (.provider | select(. | has(\"gcp\")).gcp.serviceAccountKeyPath) = \"serviceAccountKey.json\"" \
           constellation-conf.yaml
 
+        yq eval -i \
+          "(.provider | select(. | has(\"aws\")).aws.region) = \"eu-central-1\" |
+            (.provider | select(. | has(\"aws\")).aws.zone) = \"eu-central-1a\" |
+            (.provider | select(. | has(\"aws\")).aws.iamProfileControlPlane) = \"e2e_test_control_plane_instance_profile\" |
+            (.provider | select(. | has(\"aws\")).aws.iamProfileWorkerNodes) = \"e2e_test_worker_node_instance_profile\" |
+            (.provider | select(. | has(\"aws\")).aws.enforcedMeasurements) = [11,12]" \
```
[the remark this answers is not in the document]
Comment (katexochen) [reply .github/actions/constellation_create/action.yml:94]: This wouldn't work for debug images, as the PCRs differ between images.

[the remark this answers is not in the document]
Comment (3u13r) [reply .github/actions/constellation_create/action.yml:94]: Ah, we moved clusterID to PCR[15]: https://github.com/edgelesssys/constellation/commit/899ca91aa376f41617f49a588afe99f27ee2a59e 
We should do this here as well.

**Pull request merged**